pypeit/scripts/show_2dspec.py

```diff
@@ -77,6 +77,10 @@ def get_parser(cls, width=None):
                                  'flux calibration')
         parser.add_argument('--channels', type=str,
                             help='Only show a subset of the channels (0-indexed), e.g. 1,3')
+        parser.add_argument('--prefix', type=str, default='',
+                            help="Append this to the channel name [let's you do more than one set]")
+        parser.add_argument('--no_clear', default=False, action="store_true",
+                            help='Do *not* clear all existing tabs')
```

Comment:
FWIW, I try to keep the argument and parsed object name the same if I can.  So instead, I would use:
```
parser.add_argument('--no_clear', dest='clear', default=True, action="store_false",
                    help='Do *not* clear all existing tabs')
```
and then below it would be
```
clear=args.clear
```

Reply:
nice, that's a new one to me.  Done!